events/wc_dark_cult_events.txt

```diff
@@ -1688,6 +1688,11 @@ character_event = {
 			factor = 25
 			prefers_cthun_score = yes
 			stewardship_score = yes
+			
+			modifier = {
+				factor = 10
+				controls_cthun_prison_trigger = yes
+			}
```

Comment:
Why it's not inside `prefers_cthun_score`?
P.S. `stewardship_score` must be here, one scripted score can't be inside another.